2020/13xxx/CVE-2020-13674.json

```diff
@@ -1,18 +1,74 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'CWE-352 Cross-Site Request Forgery (CSRF)'}]}]}